Source/CDRStateTracker.m

```diff
@@ -0,0 +1,18 @@
+#import "CDRStateTracker.h"
+#import "CDRRunState.h"
+
+@implementation CDRStateTracker
+
+- (void)didStartBuildingTestTree {
```

Comment:
That's a good point, although this is an internal implementation that isn't intended to be exposed. This should be a project-level header with no visibility at all.

I think this should stay consistent with however we end up naming the enum, however, so I'm happy to improve this too.